.github/workflows/publish.yml

```diff
@@ -1,61 +1,61 @@
-name: Publish Docker image, Helm chart and Wire build
+name: Publish
+
 on:
   push:
-    branches: [master, dev, edge]
+    branches: [master, dev]
     tags:
       - '*staging*'
       - '*production*'
+
+concurrency:
+  group: publish-${{ github.ref }}
+  cancel-in-progress: true
+
 jobs:
   build:
     name: Build Docker image and Helm Chart
     runs-on: buildjet-8vcpu-ubuntu-2204
+
     outputs:
       wire_builds_target_branches: ${{ steps.output_target_branches.outputs.wire_builds_target_branches }}
       image_tag: ${{ steps.push_docker_image.outputs.image_tag }}
       release_name: ${{ steps.push_docker_image.outputs.release_name }}
       chart_version: ${{ steps.publish_helm_chart.outputs.chart_version }}
+      changelog: ${{ steps.generate_changelog.outputs.changelog }}
+      unit_tests_report: ${{ env.UNIT_TEST_REPORT_FILE }}
+      build_artifact: ${{ env.BUILD_ARTIFACT }}
```

Comment:
Those are artifact that are outputed by the job so that subsequent job can rely on this without having to `checkout` and `yarn` the entire codebase (and make them completely independent from the codebase)